cvat-ui/src/reducers/meta-reducer.ts

```diff
@@ -0,0 +1,46 @@
+import { MetaState } from './interfaces';
```

Comment:
``Meta`` is not an informative name. Let's find another. And why did you create new reducer & actions instead of using ``plugins`` that are dedicated for that?

Reply:
Meta means that it is the different level of abstraction for other state element. Because based on this data we decide how much functionality of application will be available.

Reply:
> Because based on this data we decide how much functionality of application will be available.

Sounds like plugins :)

It looks like there are some things to discuss. Let's do it with the team on today's meeting or have a different call.